### PR TITLE
#82 Add GradientConfig

### PR DESCRIPTION
**Pull request by chrischrischris**
* Setup js in apps/structure/app
* Add a gradient picker creator for configs

### CODE REVIEW

File: apps/structure/app/webpack.config/stats.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    publicPath: false,
+    reasons: false,
+    source: false,
+    usedExports: false,
+    version: false,
+    timings: true,
+    errors: true,
+    errorDetails: true,
+    warnings: true,
+};
```
Comment (auniverseaway) [inline apps/structure/app/webpack.config/stats.js:41]: This is located in the root of the mono repo.

Comment (chrischrischris) [reply apps/structure/app/webpack.config/stats.js]: Removed with 4d134df

**Comment by npeltier** [comment]
interesting error as well:
```[WARNING] npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})```

**Comment by chrischrischris** [comment]
@npeltier It is just a warning that happens on non-OSX platforms.
From [this link](https://github.com/google/web-starter-kit/issues/796#issuecomment-274065368):

> There is nothing to solve here. It is an optional dependency and is only needed in MacOS environments. Linux and Windows both provide native ways to do what this dependency is needed for on MacOS.

What I will try is seeing if using `npm i --no-optional` will solve the warning.

**Pull request merged**